### PR TITLE
(feat): Ability to enable and disable a survey

### PR DESCRIPTION
**Pull request by NiazMorshed2007**
This pr introduces a new feature of enabling users to activate or deactivate the survey, described in #129

### CODE REVIEW

File: src/features/surveys/managers/surveyResultsManager.ts [hunk 1]
```diff
@@ -89,6 +92,18 @@ export const useSurveyResultsManager = () => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  useEffect(() => {
+    const updateSurveyStatus = async () => {
+      await updateDoc(doc(db, 'surveys', surveyId), {
+        isActive: isSurveyActive,
+      });
```
Comment (Ryczko) [inline src/features/surveys/managers/surveyResultsManager.ts:99]: We should also handle errors here. When this action successed we should display `toast.success()` with success message and in other cases `toast.error()` with error message.

File: src/pages/survey/[surveyId]/index.tsx [hunk 2]
```diff
@@ -30,40 +31,47 @@ function AnswerPage() {
...
+                  variant={ButtonVariant.PRIMARY}
+                  sizeType={ButtonSize.MEDIUM}
+                  isLoading={isAnswering}
+                >
+                  Send
+                </Button>
+              </div>
+            </div> : <div className='container m-auto mb-6 px-4 text-center md:px-8'>
+              <h1 className='text-5xl'>🙁</h1>
+              <h1 className='mt-5 text-xl'>Oops,, Survey is no longer active</h1>
```
Comment (Ryczko) [inline src/pages/survey/[surveyId]/index.tsx:71]: remove double `,,`

File: src/features/surveys/managers/surveyAnswerManager.ts
Comment (Ryczko) [inline src/features/surveys/managers/surveyAnswerManager.ts:1]: We should also block sending answers when user opened answers page and meantime owner of survey change its status to inactive. In that case when user is sending answers we need check if it is active and if not redirect user to homepage and display `toast.error()` with information about closed survey.

File: src/pages/survey/[surveyId]/index.tsx [hunk 3]
```diff
@@ -30,40 +31,47 @@ function AnswerPage() {
...
+            </div>
```
Comment (Ryczko) [inline src/pages/survey/[surveyId]/index.tsx:72]: Maybe let's add also button that will redirect user to home page :)

**Review by Ryczko — CHANGES_REQUESTED** [review]
Thank you for your contribution to Employee Pulse. We appreciate it very much! 🥳
A few notes on your changes.

File: src/pages/survey/[surveyId]/index.tsx (hunk 3 above)
Comment (NiazMorshed2007) [reply src/pages/survey/[surveyId]/index.tsx:72]: Sure :)

**Review by Ryczko — APPROVED** [review]

**Comment by Ryczko** [comment]
Great job, thanks! 👏

**Pull request merged**